packages/dds/tree/src/simple-tree/api/viewSchemaToSimpleSchema.ts

```diff
@@ -81,34 +81,25 @@ function toSimpleNodeSchema(schema: TreeNodeSchema): SimpleNodeSchema {
 	});
 }
 
-// TODO: Use a stronger type for leaf schemas once one is available (see object schema handler for an example).
-function leafSchemaToSimpleSchema(schema: TreeNodeSchema): SimpleLeafNodeSchema {
+function copySimpleLeafSchema(schema: SimpleLeafNodeSchema): SimpleLeafNodeSchema {
```

Comment:
It would probably be worth commenting (somewhere in this file) why `toSimpleNodeSchema` needs to do these copies.

Reply:
It doesn't need them. We could write a version that does copies, or one that doesn't. The current one does so it is now documented clearly. If we want to change this behavior we could.

Reply:
I made this function able to do both for clarity, and added a bunch of docs in a few places about it.